examples/rust/ockam_kafka/examples/example.rs

```diff
@@ -0,0 +1,28 @@
+// This node routes a message, to a worker on a cloud node, over the tcp transport.
+
+use ockam::{route, Context, Result, TcpTransport, TCP};
```

Comment:
Using the Hub example for demonstration.
This file needs to contain the secure channel via streams example